src/Microsoft.DotNet.Darc/DarcLib/LocalGitClient.cs

```diff
@@ -333,4 +349,31 @@ public async Task<string> BlameLineAsync(string repoPath, string relativeFilePat
         result.ThrowIfFailed($"Failed to blame line {line} of {repoPath}{Path.DirectorySeparatorChar}{relativeFilePath}");
         return result.StandardOutput.Trim().Split(' ').First();
     }
+
+    public void AddGitAuthHeader(IList<string> args, IDictionary<string, string> envVars, string repoUri)
+    {
+        var token = _remoteConfiguration.GetTokenForUri(repoUri);
+
+        if (token == null)
+        {
+            return;
+        }
+
+        var repoType = GitRepoUrlParser.ParseTypeFromUri(repoUri);
+        if (repoType == GitRepoType.None)
+        {
+            return;
+        }
+
+        const string ENV_VAR_NAME = "GIT_REMOTE_PAT";
```

Comment:
Are there any docs for this? If it were to brake in the future, would be nice if the header definition had some sort of a helpful link pointing the developer to a place where they could check if the header did not change

Reply:
This is our custom string - just a name of an env git should get this from. It's not the name of the AzDO header.

Reply:
This is documented with the usages of the `--config-env` git option